bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookKeeperServerStats.java

```diff
@@ -42,6 +42,7 @@
     public final static String BOOKIE_ADD_ENTRY = "BOOKIE_ADD_ENTRY";
     public final static String BOOKIE_RECOVERY_ADD_ENTRY = "BOOKIE_RECOVERY_ADD_ENTRY";
     public final static String BOOKIE_READ_ENTRY = "BOOKIE_READ_ENTRY";
+    public final static String BOOKIE_READ_LAST_CONFIRMED = "BOOKIE_READ_LAST_CONFIRMED";
```

Comment:
this new stat is not related to the issue.
for me it is fine anyway, but I wonder if we have to commit it in a separate issue